clusters/build-clusters/01_cluster/ci-search/deploy.yaml

```diff
@@ -4,164 +4,72 @@ metadata:
   name: ci-search
 
 ---
-# Import the ImageStream from CI
-kind: ImageStream
-apiVersion: image.openshift.io/v1
+apiVersion: route.openshift.io/v1
+kind: Route
 metadata:
   name: ci-search
   namespace: ci-search
 spec:
-  lookupPolicy:
-    local: true
-  tags:
-  - from:
-      kind: DockerImage
-      name: registry.ci.openshift.org/ci/ci-search:latest
-    importPolicy:
-      scheduled: true
-    name: latest
-    referencePolicy:
-      type: Local
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  to:
+    kind: Service
+    name: ci-search-redirect
 
-# deploy the controller
 ---
-kind: Service
 apiVersion: v1
+kind: ConfigMap
 metadata:
-  name: search
+  name: ci-search-redirect-config
   namespace: ci-search
-spec:
-  selector:
-    app: search
-  ports:
-  - port: 80
-    targetPort: 8080
+data:
+  default.conf: "server {\n  listen 8080;\n  return 302 https://search.dptools.openshift.org$request_uri;\n}\n"
 
 ---
-apiVersion: policy/v1
-kind: PodDisruptionBudget
-metadata:
-  name: ci-search
-  namespace: ci-search
-spec:
-  minAvailable: 1
-  selector:
-    matchLabels:
-      app: search
----
-kind: StatefulSet
 apiVersion: apps/v1
+kind: Deployment
 metadata:
-  name: search
+  labels:
+    app: ci-search-redirect
+  name: ci-search-redirect
   namespace: ci-search
-  annotations:
-    image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"ci-search:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"web\")].image"}]'
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
This should probably be `2` to field as many requests as possible

Reply:
added in https://github.com/openshift/release/pull/49035/commits/7ef14e5d1c293c7e05b8bdfe2460684964e42efb